style/haskell/README.md

```diff
@@ -5,13 +5,16 @@ Haskell
 
 * Break long expressions before operators or keywords.
 * Break long type signatures between arguments.
+* Don't use a blank line between pragma and module statements.
```

Comment:
I couldn't find any code on GitHub that formats pragma statements without a subsequent blank line. What code are you looking at?

(On GitHub search results, ghc was the biggest user of language pragmas.)